File: src/ProgrammingFundamentals/lab/01-01-02/02.cpp
#include <iostream>

using namespace std;

int main() {
	int a, b, c, //Користувацькі значення
	x1, x2, //Інтервал
	h; //Крок
	double result;

	cout << "Введіть число a: ";
	cin >> a;
	cout << "Введіть число b: ";
	cin >> b;
	cout << "Введіть число c: ";
	cin >> c;
	cout << "Введіть число x1: ";
	cin >> x1;
	cout << "Введіть число x2: ";
	cin >> x2;
	cout << "Введіть число h: ";
	cin >> h;


	for (int x = x1; x <= x2; x += h) {
		if ((x + 5) < 0 && c == 0) {
			result = (1 / (a * x)) - b;
		} else if ((x + 5) > 0 && c != 0) {
			result = (x - a) / x;
		} else {
			result = (10 * x) / (c - 4);
		}
		if (!((a != 0 && b != 0) || (b != 0 && c != 0))) {
			result = (int) result;
		}
		cout << result << endl;
	}
}

File: webdesign/modules/01-01-02/02.js
//Послідовність Фібоначчі  визначається так: a0 = 0, a1 = 1, ak = ak-1 + ak-2 при k>1. Дане n, обчислити an.
var n = 12,
	result = fib(n);

function fib(n) {
	return n < 2 ? n : fib(n - 1) + fib(n - 2);
}

console.log(result);